### DISCUSSION TITLE
发表及编辑文章时 自动生成的文章更新时间为服务器时间

### DISCUSSION THREAD

**Discussion by 37748846** [Qexo/Qexo#354] [General]
发表及编辑文章时 自动生成的文章更新时间为服务器时间 ，qexo部署在vercel
![QSITZ)H6L$(_2D8BB GG66V](https://user-images.githubusercontent.com/39357514/231097947-930efafd-227a-4865-868f-fda6a99a6e67.png)
发布时间，更新时间差8小时，忘记手动修改的话会造成更新时间小于发布时间，很诡异
搜索和说明中没有找到相关问题，请帮忙接答 谢谢！ @am-abudu 
加qq群了 好几天没有通过
又遇到问题了 升级了2.7后
markdown的编辑栏不显示
![$~M (6 LX51{GH5 H2_4{O3](https://user-images.githubusercontent.com/39357514/231103590-4de60817-145d-4fd9-a17b-65ecf3e93f9f.png)

**Comment by am-abudu** [comment]
已注明GTM(格林尼治时间)，程序会自动识别并转化，无须担心

**Reply by am-abudu** [reply]
不显示可能是网络问题，可以F12检查一下资源加载情况

**Reply by 37748846** [reply]
上一问题谢谢解答
![image](https://user-images.githubusercontent.com/39357514/231160315-e020a12b-da19-46f4-8c23-28df26c25cd7.png)
cdn都加载不上
是不是多了个.map?

**Reply by am-abudu** [reply]
我这里一切正常，如果没有更多人的报告将视为你的环境问题

**Reply by 37748846** [reply]
应该是我的问题，换了台电脑试了下，没问题，麻烦了，谢谢

**Reply by 37748846** [reply]
官方sm.ms的图床配置例子介绍上，apikey的双引号前多了一个空格,完全复制介绍的代码会造成错误.

**Reply by 37748846** [reply]
`<time title="创建时间：2023-04-11 22:49:40 / 修改时间：14:49:39" itemprop="dateCreated datePublished" datetime="2023-04-11T22:49:40+08:00">2023-04-11</time>`
我用的hexo 部署在vercel 我把hexo的config编辑时区为上海.上边是datetime+8，update还是格林威治时间 页面显示修改时间早于创建时间,程序没有自动转换

**Reply by am-abudu** [reply]
使用Dev分支
